### DISCUSSION TITLE
Using async-alpine with astro

### DISCUSSION THREAD

**Discussion by willbocc** [Accudio/async-alpine#30] [Show and tell]
I recently built this sustainability report using astro as static site build tool and alpine for interactivity: https://responsibilityreport2022.ganni.com/

**Comment by Accudio** [comment]
Nice website, it looks great!

Would you be happy for me to put it in the showcase on the Async Alpine website?

**Reply by willbocc** [reply]
Thanks, and yes of course!

**Reply by Accudio** [reply]
I've added the site to the Showcase at [async-alpine.dev/showcase](https://async-alpine.dev/showcase/), which also displays three random items on the homepage and refreshes each day. [async-alpine.dev](https://async-alpine.dev)

Thanks!
